src/Adapter/Product/Image/Repository/ProductImageRepository.php

```diff
@@ -149,6 +149,20 @@ public function getImagesIds(ProductId $productId): array
         return $imagesIds;
     }
 
+    /**
+     * @param ProductId $productId
+     *
+     * @return ImageId|null
+     *
+     * @throws CoreException
+     */
+    public function getDefaultImageId(ProductId $productId): ?ImageId
```

Comment:
Could you add unit test for this method ?

Reply:
As I already argued there is nearly no interest in **unit** testing a repository, an integration or behaviour test has much more values

I won't test this class directly, however, it will be tested indirectly via the behat test of the command using it